calibration/NUTNRM/CGINS-NUTNRM-00661__20220624.csv

```diff
@@ -0,0 +1,8 @@
+661,CC_lower_wavelength_limit_for_spectra_fit,217,
+661,CC_upper_wavelength_limit_for_spectra_fit,240,
+661,CC_wl,"[188.96, 189.76, 190.56, 191.35, 192.15, 192.95, 193.75, 194.55, 195.35, 196.15, 196.95, 197.76, 198.56, 199.36, 200.16, 200.96, 201.76, 202.56, 203.37, 204.17, 204.97, 205.78, 206.58, 207.38, 208.18, 208.99, 209.79, 210.6, 211.4, 212.2, 213.01, 213.81, 214.62, 215.42, 216.23, 217.03, 217.84, 218.64, 219.45, 220.26, 221.06, 221.87, 222.67, 223.48, 224.29, 225.09, 225.9, 226.71, 227.52, 228.32, 229.13, 229.94, 230.75, 231.55, 232.36, 233.17, 233.98, 234.79, 235.59, 236.4, 237.21, 238.02, 238.83, 239.64, 240.45, 241.26, 242.07, 242.88, 243.69, 244.49, 245.3, 246.11, 246.92, 247.73, 248.54, 249.35, 250.16, 250.98, 251.79, 252.6, 253.41, 254.22, 255.03, 255.84, 256.65, 257.46, 258.27, 259.08, 259.89, 260.71, 261.52, 262.33, 263.14, 263.95, 264.76, 265.57, 266.39, 267.2, 268.01, 268.82, 269.63, 270.44, 271.26, 272.07, 272.88, 273.69, 274.5, 275.32, 276.13, 276.94, 277.75, 278.56, 279.38, 280.19, 281, 281.81, 282.62, 283.44, 284.25, 285.06, 285.87, 286.68, 287.5, 288.31, 289.12, 289.93, 290.75, 291.56, 292.37, 293.18, 293.99, 294.81, 295.62, 296.43, 297.24, 298.05, 298.87, 299.68, 300.49, 301.3, 302.11, 302.92, 303.74, 304.55, 305.36, 306.17, 306.98, 307.79, 308.61, 309.42, 310.23, 311.04, 311.85, 312.66, 313.47, 314.28, 315.1, 315.91, 316.72, 317.53, 318.34, 319.15, 319.96, 320.77, 321.58, 322.39, 323.2, 324.01, 324.82, 325.63, 326.44, 327.25, 328.06, 328.87, 329.68, 330.49, 331.3, 332.11, 332.92, 333.73, 334.54, 335.34, 336.15, 336.96, 337.77, 338.58, 339.39, 340.19, 341, 341.81, 342.62, 343.43, 344.23, 345.04, 345.85, 346.65, 347.46, 348.27, 349.07, 349.88, 350.69, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.13, 357.94, 358.74, 359.55, 360.35, 361.15, 361.96, 362.76, 363.57, 364.37, 365.17, 365.98, 366.78, 367.58, 368.38, 369.19, 369.99, 370.79, 371.59, 372.39, 373.2, 374, 374.8, 375.6, 376.4, 377.2, 378, 378.8, 379.6, 380.4, 381.2, 382, 382.8, 383.6, 384.39, 385.19, 385.99, 386.79, 387.59, 388.38, 389.18, 389.98, 390.77, 391.57, 392.37, 393.16, 393.96, 394.75]",
```

Comment:
All the values look good. I can't remember if there is a reason why some nutnrs have NTR-(S/N) while others just have their serial number with no NTR prefixed. Thanks!